src/cli/commands/force/lightning/lwc/test/ui/configure/mobile.ts

```diff
@@ -0,0 +1,462 @@
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { flags, SfdxCommand } from '@salesforce/command';
+import { Logger, Messages, SfError } from '@salesforce/core';
+import { AndroidEnvironmentRequirements } from '@salesforce/lwc-dev-mobile-core/lib/common/AndroidEnvironmentRequirements';
+import { AndroidVirtualDevice } from '@salesforce/lwc-dev-mobile-core/lib/common/AndroidTypes';
+import { AndroidUtils } from '@salesforce/lwc-dev-mobile-core/lib/common/AndroidUtils';
+import {
+    CommandLineUtils,
+    FlagsConfigType
+} from '@salesforce/lwc-dev-mobile-core/lib/common/Common';
+import { CommonUtils } from '@salesforce/lwc-dev-mobile-core/lib/common/CommonUtils';
+import { IOSEnvironmentRequirements } from '@salesforce/lwc-dev-mobile-core/lib/common/IOSEnvironmentRequirements';
+import { IOSSimulatorDevice } from '@salesforce/lwc-dev-mobile-core/lib/common/IOSTypes';
+import { IOSUtils } from '@salesforce/lwc-dev-mobile-core/lib/common/IOSUtils';
+import {
+    RequirementProcessor,
+    HasRequirements,
+    CommandRequirements
+} from '@salesforce/lwc-dev-mobile-core/lib/common/Requirements';
+import dedent from 'ts-dedent';
+import util from 'util';
+
+// Initialize Messages with the current plugin directory
+Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = Messages.loadMessages(
+    '@salesforce/lwc-dev-mobile',
+    'test-ui-configure-mobile'
+);
+
+export class Mobile extends SfdxCommand implements HasRequirements {
+    public static description = messages.getMessage('commandDescription');
+
+    public static examples = [
+        `$ force:lightning:lwc:test:ui:configure:mobile -p iOS -d sfdxdebug -o './wdio.conf.js' -f jasmine -r local -t 4723 -u 'http://localhost' -i '/path/to/myPageObjects.config.json' -b '/path/to/my.app'`,
+        `$ force:lightning:lwc:test:ui:configure:mobile -p Android -d sfdxdebug -o './wdio.conf.js' -f jasmine -r local -t 4723 -u 'http://localhost' -i '/path/to/myPageObjects.config.json' -b '/path/to/my.apk' -a .MainActivity -k com.example.android.myApp`
+    ];
+
+    public static defaultOutputFile = './wdio.conf.js';
+    public static supportedTestFrameworks = ['jasmine', 'mocha', 'cucumber'];
+    public static supportedTestRunners = ['local', 'browser'];
+    public static defaultTestRunnerBaseUrl = 'http://localhost';
+
+    private static createError(stringId: string): SfError {
+        return new SfError(
+            messages.getMessage(stringId),
+            'lwc-dev-mobile',
+            Mobile.examples
+        );
+    }
+
+    public static flagsConfig = {
+        ...CommandLineUtils.createFlagConfig(FlagsConfigType.Platform, true),
+        devicename: flags.string({
+            char: 'd',
```

Comment:
Instead of assigning a single character for all flags (i.e --devicename and -d) perhaps we should only assign single chars for some flags? For example as you can see [here](http://beta.webdriver.io/docs/clioptions.html), some flags have don't take single chars.

We have `-p = platform` for all of our command, which now conflicts with `-p = port`, so that's why I'm thinking that perhaps we should not use single chars for everything? (in this PR I'm using `-t = port` for now)

Reply:
I can go with that as long as oclif allows that.